README.md

```diff
@@ -1,4 +1,4 @@
-# Twitter Clone
+# Twitter Clone - cange for pull request
```

Comment:
Lots of pulls / push changes.  Good to show the timeline of amendments & easier to review.  Nice!